wiki/Multi/en.md

```diff
@@ -0,0 +1,487 @@
+
+To enter a Match Setup, click on one of the desired row.
+
+**Note:** If the game mode symbol contains a lock icon, clicking on it will prompt an empty text box to enter the Match Setup password.
+The player should enter the correct password to access the locked Match Setup.
+Alternatively, the player can request an invite from the Match Setup's host directly to bypass the locked password requirement.
+
+---
+
+A new Match Setup can be created by pressing the `New Game` button.
+
+The `Quick Join` button will direct the player to an available Match Setup randomly based on current rank range.
+
+Click on the `Back to Menu` button to return to the main menu.
+
+## New game
+
+![New Game setup screen][Multi_new_game image]
+
+_Setting up a new game_
+
+Title | Description
+:---: | :---
+**Game Name**   | **Name of the room.** Default to `{accountname}`'s game. Foreign characters allowed (will be parsed). To make the Match Setup private, enable the `Require Password to join` button.
+**Password**    | **Password to enter the room.** Only appears if the `Require Password to join` button was enabled. Match Setup game mode symbol will be shown with a lock.
+**Max players** | **Total number of players allowed to join the Match Setup.** Default to 8 players, with range from 2 to a total of 16 players, including the player as host. It can be adjusted later by locking/unlocking the placements.
+
+Press the `1. Start Game` button to create the room with the player as the host of the room and the beatmap playing in the jukebox as the match's beatmap difficulty.
+
+The player will be directed to **Match Setup**.
+
+## Match Setup
+
+<!-- One/Two image here? Subsections of the images later like what was done in Interface of the old osu!wiki for Song Selection? -->
+
+![Host room view][Multi_roomhost image]
+
+_Host room view_
+
+![Player room view][Multi_roomplayer image]
+
+_Player room view_
+
+The top section at the left display the name of the screen, status in the room (either host or player), player box at the middle based on current game mode played, and a pause button to pause the beatmap's song (if the player has the beatmap) located at the right side.
+
+---
+
+The bottom section is the [Chat Console][Chat Console wikilink].
+A new tab called `#multiplayer` will be opened, with [BanchoBot][BanchoBot wikilink] providing the match history link in the first line (Match history available [here][Match History link]).
+The link is <http://osu.ppy.sh/mp/x> (replace "x" with digits generated by Bancho for this Match Setup), and will be removed after one month.
+
+After a concluded match, result will be displayed at another tab called `#userlog` privately (only apply to Head to Head Team Mode).
+It should be sent by [BanchoBot][BanchoBot wikilink] with content, `You placed #{placement} of {total} players in multiplayer on {Beatmap} [difficulty] <game mode>.`
```

Comment:
What should be sent by BanchoBot (The sentence started with `It` but what is it)?